scripts/pi-hole/php/password.php

```diff
@@ -40,6 +40,13 @@
             if($postinput == $pwhash)
             {
                 $_SESSION["hash"] = $pwhash;
+
+                // Login successful, redirect the user to the homepage to discard the POST request
+                if ($_SERVER['REQUEST_METHOD'] === 'POST' && $_SERVER['QUERY_STRING'] === 'login') {
+                    header(sprintf('Location: index.php'));
```

Comment:
What's the reason for not using `sprintf()` here?

How about
```php
header("Location: index.php)";
```
?

Reply:
Because it's faster

Reply:
Nah, I'm just kidding, I was playing with the possibility of redirecting the user to the page he came from with the REFERER and forgot to remove the sprintf.